server/src/main/scala/ch.epfl.scala.index.server/routes/api/impl/PublishProcess.scala

```diff
@@ -82,7 +80,7 @@ private[api] class PublishProcess(paths: DataPaths, dataRepository: ESRepo, db:
 
                   Meta.append(
                     paths,
-                    Meta(data.hash, data.path, data.created),
+                    Meta(data.hash, data.path, data.datetimeCreated),
```

Comment:
Why not change that to `Instant` and remove the dependency to `joda` here?

Reply:
we cannot, this is what we discussed yesterday. The file is written with datetime.

Reply:
ok